executor/admin.go

```diff
@@ -415,6 +437,11 @@ func (e *RecoverIndexExec) backfillIndexInTxn(ctx context.Context, txn kv.Transa
 		return result, err
 	}
 
+	recoveringClusteredIndex := e.index.Meta().Primary && e.table.Meta().IsCommonHandle
```

Comment:
We can return much earlier if we are recovering clustered index primary key.

Reply:
Updated.